src/yadism/coefficient_functions/coupling_constants.py

```diff
@@ -2,9 +2,12 @@
 import logging
 
 import numpy as np
+from eko import basis_rotation as br
 
 logger = logging.getLogger(__name__)
 
+flavors = br.quark_names
```

Comment:
We could use directly `br.quark_names`, without reexporting from here.

Reply:
I know - but I was just lazy, as searching for "flavors" inside yadism is not exactly the best idea ever ;-)

Reply:
One more good reason to get rid of it... but afterwards!